analyses/molecular-subtyping-ATRT/01-ATRT-molecular-subtyping-data-prep.Rmd

```diff
@@ -168,14 +176,19 @@ metadata <- metadata %>%
     age_at_diagnosis_days,
     germline_sex_estimate,
     primary_site
-  )
+  ) %>%
+  dplyr::group_by(
+    sample_id,
+    Kids_First_Participant_ID
+  ) %>%
+  dplyr::summarize_all(function(x) paste(sort(unique(x)), collapse = ", "))
```

Comment:
@cbethell if we do this collapse biospecimen IDs step up front, we can drop the biospecimen ID columns as we go since we join on `sample_id`. This means we don't have to collapse at the end and run the risk of having a bunch of duplicated columns.